src/mibase.ts

```diff
@@ -706,6 +706,15 @@ export class MI2DebugSession extends DebugSession {
 	}
 
 	protected override evaluateRequest(response: DebugProtocol.EvaluateResponse, args: DebugProtocol.EvaluateArguments): void {
+		// TODO: evaluate the expression in the global scope
+		if (!args.frameId) {
+			response.body = {
+				variablesReference: 0,
+				result: ""
+			};
+			this.sendResponse(response);
```

Comment:
sendErrorResponse instead of plain sendResponse

Reply:
done